include/localization.js

```diff
@@ -35,8 +35,8 @@ module.exports = function LocalizationModule(pb) {
      * @constructor
      * @param {Object} request The request object
      */
-    function Localization(request){
-        
+    function Localization(request, activeTheme){
+        this.activeTheme = activeTheme;
```

Comment:
If the setter implementation is used you will not need the extra parameter on the constructor.  However, I would go ahead and set the default value on the property to ```pencilblue```